resources/dicts/thoughts/alive/general.json

```diff
@@ -1100,63 +1123,61 @@
             "playful"
         ]
     },
-	{
-        "id": "playful_hunter",
+    {
+        "id": "rebellious_gen",
         "thoughts": [
-		"Stuck {PRONOUN/m_c/poss} tongue out at a mouse that escaped {PRONOUN/m_c/poss} pounce",
```

Comment:
seems like you lost some playful thoughts? Was this intentional?

Reply:
they're actually up next to the childish thoughts! i noticed the group here was identical to one that childish has, and it makes just as much sense for childish cats to play with kits as it does for playful cats, so i merged them!
(they're up next to childish since childish is alphabetically the first trait in the constraint, in case you were wondering :0)